core/templates/dev/head/pages/exploration_player/exploration_footer_directive.html

```diff
@@ -193,6 +215,10 @@
         </li>
       </ul>
       <hint-and-solution-buttons ng-if="windowIsNarrow"></hint-and-solution-buttons>
+      <div class="star-container" ng-if="windowIsNarrow">
+          <i class="material-icons star">star_rate</i>
+        <span class="star-content" ng-bind-html="correctCount()"></span>
```

Comment:
Why use ng-bind-html instead of just {{...}} between the spans?

If your content is not html in the first place then ng-bind-html is too heavy.

Reply:
Done!